tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -78,7 +78,143 @@ void set_log_callback()
 #endif
 }
 
-oe_result_t gen_report(oe_enclave_t* enclave)
+OE_INLINE uint16_t ReadUint16(const uint8_t* p)
```

Comment:
coding guidelines say to use snake_case in general... this file should at least try to be internally consistent in style.